### ISSUE TITLE
Estimation of Binding Capacity and KM

### ISSUE THREAD

**Issue by JensBorghardt** [Open-Systems-Pharmacology/Forum#63]
Hi everyone,
I have a question related to a parameter identification problem. For the compound of interest, I investigate whether saturable binding to a specific binding partner can explain its unexpected high terminal half-life. Therefore, I included a binding partner in the individual block and defined the respective KM/koff for the compound. The result can be seen below:
 
![image](https://user-images.githubusercontent.com/25690678/29463996-ce5f01ac-8434-11e7-8fc9-cd9079ceeddb.png)

Here, you can see that I have different species; the complex, the unbound drug, and the free binding partner. What we typically measure as plasma concentrations would in contrast be the sum of compound in the complex + the unbound compound. Unfortunately, I do not know wether there exists the opportunity to directly display a combined profile (bound + unbound) of the compound? Furthermore, my next aim would be to adequately estimate the parameter KM and the expression of the binding partner with the parameter estimation. However, I can not directly select the bound + unbound sum as an output, which however would be required for an adequate estimation. Do you know a workaround for the problem or do I have to solve this problem in MoBi? 

Thanks for your help and best regards,
Jens

**Comment by MarKrauss** [comment]
Hi Jens,
user-defined observers can be quite straightforward created in MoBi, but not in PK-Sim. After creation of the Observer "BoundPlusUnbound", the Observer can also be used for Parameter Identification in MoBi, which is the same as in PK-Sim, so handling shouldn't be a problem. 
I attached an example project where I added a Binding Partner X as a possible target for Midazolam (just an example!) with the observer that sums up the concentration of Midazolam and Midazolam bound to Binding Partner X (MoBi File). 
Hope this helps, best
Markus

[Example_BoundUnboundObserver.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/1241435/Example_BoundUnboundObserver.zip)
